docker-compose-prod.yml

```diff
@@ -0,0 +1,63 @@
+# Version 3, most current, and recommended compose file version
+# https://docs.docker.com/compose/compose-file/compose-versioning/
+version: '3'
+
+services:
+  # First service
+  telescope:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: .env
+    environment:
+      # a place-holder for the variable passed by CLI with a default value
+      # the passed or default value is a command to run `telescope`
+      - script=${script:-start}
+      # override the .env variable for `REDIS_URL`
+      - REDIS_URL=redis://redis:${REDIS_PORT}
+    depends_on:
+      - redis
+    ports:
+      - '$PORT:$PORT'
+    restart: always
+
+  # Second service
+  redis:
+    image: redis:latest
+    ports:
+      - '$REDIS_PORT:$REDIS_PORT'
+
+  # Third service
+  # SSO Identity Provider test service, https://simplesamlphp.org
+  # Access to the authentication page via http://localhost:8080/simplesaml or https://localhost:8443/simplesaml
+  ssoipd:
```

Comment:
I wonder if we should rename this to `login`, which is more recongnizable to developers who don't fully understand how SAML works (SSO IPD is right, but not an obvious name).

Reply:
We might also end up with 3 of these files: dev, staging, and prod.  In prod we won't want the login stuff, since we'll use Seneca's SSO server.

Reply:
Since we're going to have 3 these, maybe I should rename the new one added in this PR to `docker-compose-stage.yml`?